### PR TITLE
Patch unit test 1

### PR DESCRIPTION
**Pull request by indcoder**

### CODE REVIEW

**Comment by qthanh111** [comment]
[](url)

File: test/test-homepage.js [hunk 1]
```diff
@@ -9,7 +9,7 @@ describe("check build output for homepage", () => {
     const FILENAME = "_site/index.html";
 
     if (!existsSync(FILENAME)) {
-      it("WARNING skipping tests because FILENAME does not exist", () => {});
+      it("WARNING skipping tests because FILENAME does not exist", () => { });
```
Comment (cramforce) [inline test/test-homepage.js:12]: Please revert

**Comment by indcoder** [comment]
Once again, the linter strikes.

Btw, what linting rules are being used in the project....because the markdown files on my VS Code are awashed in red with markdownlint errors/warnings.

**Review by cramforce — APPROVED** [review]
Thanks

**Pull request merged**